pkg/display/screen.go

```diff
@@ -131,6 +131,12 @@ func (s *Screen) Clear() {
 	s.buffer = make([]string, s.Height, s.Height)
 }
 
+// Erase clears and re-draws the screen.
+func (s *Screen) Erase() {
```

Comment:
If this pair of calls is only needed in one place (per other comment), then it probably doesn't warrant a dedicated function.